presto-spark-base/src/main/java/com/facebook/presto/spark/execution/HttpRemoteTaskResultFetcher.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution;
+
+import com.facebook.presto.execution.TaskId;
+import com.facebook.presto.spark.execution.http.PagesResponse;
+import com.facebook.presto.spark.execution.http.PrestoSparkHttpWorkerClient;
+import com.facebook.presto.spi.HostAddress;
+import com.facebook.presto.spi.PrestoException;
+import com.facebook.presto.spi.page.SerializedPage;
+import io.airlift.units.DataSize;
+import io.airlift.units.Duration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
+import static com.facebook.presto.spi.StandardErrorCode.SERIALIZED_PAGE_CHECKSUM_ERROR;
+import static com.facebook.presto.spi.page.PagesSerdeUtil.isChecksumValid;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public class HttpRemoteTaskResultFetcher
+{
+    private static final Duration FETCH_INTERVAL = new Duration(200,
+            TimeUnit.MILLISECONDS);
```

Comment:
same line;
also, can this be configurable?

Reply:
Yeah this will be configurable after we add config populators to the codebase. Right now we set constant here. There will be properties refactoring after we have populator support (A mechanism that populates all configs to local config files)